docs/octopus-cloud/index.md

```diff
@@ -6,3 +6,35 @@ description: How to work with Octopus Cloud.
 **Octopus Cloud** is the hosted version of Octopus Deploy and it has been publicly available since February 2018. We designed Octopus Cloud and self-hosted Octopus to provide the same functionality; however, there are some minor differences, for instance, with Octopus Cloud, we're [responsible](/docs/administration/security/index.md#responsibility) for taking backups, automatically upgrading the service, and maintaining and monitoring the underlying systems, and for security reasons some of the configuration and diagnostic functionality has been limited. <!-- do we have specifics? -->
 
 You can sign up for Octopus Cloud at [account.octopus.com/register](https://account.octopus.com/register), check out the [Octopus Cloud FAQ](https://octopus.com/blog/octopus-cloud-faq) blog post or the [pricing page](https://octopus.com/pricing/cloud).
+
+
+## Getting Started with Octopus Cloud
+
+Before you can start an Octopus Cloud trial, you'll need an Octopus account.
+
+You can sign up for an account at: [account.octopus.com/register](https://account.octopus.com/register).
+
+## Create an Octopus Account
+
+An Octopus account lets you manage your instances of Octopus Cloud. <!-- is there anything else we want to say about this? -->
```

Comment:
@Waldo000000 @thekatstevens is there anything we'd like to say about Octopus Accounts? I believe managing your cloud instance is everything at the moment, but is that right?

Reply:
@wordlee I believe that's right at the moment, yep, nothing else I'd like us to add :+1: